src/Entities/Entity.cpp

```diff
@@ -21,6 +21,8 @@
 
 UInt32 cEntity::m_EntityCount = 0;
 cCriticalSection cEntity::m_CSCount;
+const float cEntity::DEFAULT_GRAVITY = -9.81f;  // Meters per second per second
+const double cEntity::EPS = 0.0000001;
```

Comment:
I'm no sure about these either. First of all why isn't the gravity defined for the cWorld class? And secondly, could you possibly change EPS to the full name EPSYLON? Characters are cheap 😄